src/idsk/components/in-page-navigation/template.njk

```diff
@@ -6,7 +6,7 @@
   <div class="idsk-in-page-navigation-wrapper">
     <div class="idsk-in-page-navigation__title">Obsah</div>
     <div class="idsk-in-page-navigation__link-panel">
-      <button class="idsk-in-page-navigation__link-panel-button">{{ params.items[0].text }}</button>
+      <button aria-expanded="false" data-text-for-show="Akordeón {{ params.items[0].text }} je otvorený" data-text-for-hide="Akordeón {{ params.items[0].text }} je zatvorený" aria-label="Akordeón {{ params.items[0].text }} je zatvorený" class="idsk-in-page-navigation__link-panel-button">{{ params.items[0].text }}</button>
```

Comment:
V komponente by nemali byť priamo vložené texty, keďže potom by nešli jazykové mutácie. Pozri ako to ma napríklad table-filter